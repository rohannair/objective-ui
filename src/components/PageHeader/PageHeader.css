@import '../../styles/globals';
@import '../../styles/mixins';
@import '../../styles/layout';

.PageHeader {
  @extend row;
  @extend controlBar;
}

.children {
  display: flex;
  flex-direction: row;
  margin-left: auto;
}
