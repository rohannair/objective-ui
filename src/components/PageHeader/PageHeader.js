import React, { PropTypes } from 'react'
import styles from './PageHeader.css'

const PageHeader = p => (
  <div className={styles.PageHeader}>
    <h2>{ p.title }</h2>
    <div className={styles.children}>
      { p.children }
    </div>
  </div>
)

export default PageHeader
