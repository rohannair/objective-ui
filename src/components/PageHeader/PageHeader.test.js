import React from 'react'
import PageHeader from './index.js'

describe('Test suite for PageHeader component', () => {
  it('PageHeader should exist', () => {
    let wrapper = shallow(<PageHeader/>)
    expect(wrapper).toMatchSnapshot()
  })
})

