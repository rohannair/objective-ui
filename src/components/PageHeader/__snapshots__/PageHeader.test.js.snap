exports[`Test suite for PageHeader component PageHeader should exist 1`] = `
<div>
  <h2 />
  <div />
</div>
`;
